apps/web/components/Embed.tsx

```diff
@@ -687,7 +730,14 @@ ${getEmbedTypeSpecificString({ embedFramework: "HTML", embedType, calLink, previ
 
 /* If you are using npm */
 // npm install @calcom/embed-react
-${getEmbedTypeSpecificString({ embedFramework: "react", embedType, calLink, previewState })}
+${getEmbedTypeSpecificString({
```

Comment:
Thought of making it a hook. But embedFramework is conditional and using a hook with conditional would require me to make a state which is unnecessary. So, keeping it as a regular function only